content/en/os/1.13.x/update/guidelines.markdown

```diff
@@ -0,0 +1,62 @@
+# Guidelines for Updating In-Place
+
+## Introduction
+
+There are some guidelines and considerations to keep in mind when updating your Bottlerocket nodes _in-place_.
+Some movements between versions are updates, some are not updates, and there are some caveats you may need to consider based on the history of the project.
+
+## Update Paths
+
+Certain movements between releases of Bottlerocket are accessible through an _in-place_ update path.
+
+### Between Minor or Patch Versions of the Same Variant
+
+With the exception of major versions, it is possible to update from one released version of a variant to another released version of the same variant.
```

Comment:
- 'With the exception of major versions' -> 'With the exception of moving between major versions'
- 'it is possible to update' -> 'it is possible to update in-place'

Reply:
Instead of adding a line in 23 about the caveat you can add it in the first clause on this line - something like

"....major versions and a caveat discussed below, ..." (linked of course)